nbgitpuller/pull.py

```diff
@@ -304,13 +304,14 @@ def main():
     parser = argparse.ArgumentParser(description='Synchronizes a github repository with a local repository.')
     parser.add_argument('git_url', help='Url of the repo to sync')
     parser.add_argument('branch_name', default=None, help='Branch of repo to sync', nargs='?')
-    parser.add_argument('repo_dir', default='.', help='Path to clone repo under', nargs='?')
+    parser.add_argument('--target-dir', default='.', help='Path to clone repo under')
```

Comment:
Deleting the positional argument repo_dir seems like a breaking change. Is the idea that it is replaced with the optional `--target-dir`?

Reply:
yes, this is from a previous PR though. Here is the discussion: comment(https://github.com/jupyterhub/nbgitpuller/pull/192#discussion_r658871096). I blew this. It came through a merge from main and I don't think I pushed it into #192 as I thought I did. I can back it out of this PR and get it into the other. I think, from where things were left off in #192 we did not want to merge this change until there was a major version release because it would be breaking.